### PR TITLE
Fix small capitalization and grammar issues in docs

### PR DESCRIPTION
**Pull request by brainwane**
Checklist: (I think none of the checklist items are relevant.)

### CODE REVIEW

File: src/maintainer/infrastructure.rst [hunk 1]
```diff
@@ -29,7 +29,7 @@ Smithy contains maintenance code for conda-forge, which is used by the ``conda s
 Web services
 ------------
 
-The heroku app providing the conda-forge web services are living in `conda-forge/conda-forge-webservices <https://github.com/conda-forge/conda-forge-webservices>`_.
+The Heroku app providing the conda-forge web services is living in `conda-forge/conda-forge-webservices <https://github.com/conda-forge/conda-forge-webservices>`_.
```
Comment (scopatz) [inline src/maintainer/infrastructure.rst]: shouldn't this be "lives in"

**Comment by scopatz** [comment]
One minor suggestion! Thanks!

File: src/maintainer/infrastructure.rst (hunk 1 above)
Comment (brainwane) [reply src/maintainer/infrastructure.rst]: Thanks - fixed.

**Comment by brainwane** [comment]
Thanks! Resolved; ready for re-review.

**Comment by scopatz** [comment]
Thanks @brainwane!

**Pull request merged**